web/components/input/buttons/RadioGroup/RadioButton.sass

```diff
@@ -0,0 +1,54 @@
+.radio-button
+  margin-left: 20px;
+  font-family: Apercu Pro;
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 23px;
+  color: $primary;
+  position: relative;
+  margin-bottom: 8px;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+  &:hover input:enabled ~ .radio-img
+    background-image: url('../assets/radio/hover.svg');
+
+  input[type="radio"]
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+
+    &:checked ~ .radio-img
+      background-image: url('../assets/radio/selected.svg');
+      &:after
+        display: block;
+
+    &:disabled
+      cursor: not-allowed;
+      opacity: 0.4;
+      &~ .radio-img
+        cursor: not-allowed;
+        opacity: 0.4;
+        &~ .label-text
+          cursor: not-allowed;
+          opacity: 0.4;
+
+  .radio-img
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 24px;
+    width: 24px;
+    background-image: url('../assets/radio/unselected.svg')
+
+    &:after
+      content: "";
+      position: absolute;
+      display: none;
+  
+  .label-text
+    padding-left: 34px;
+    white-space: nowrap;
```

Comment:
can we get newlines on the end of files? 😛 

https://marketplace.visualstudio.com/items?itemName=jdforsythe.add-new-line-to-files